### ISSUE TITLE
Icons not properly loading, even after installing fonts

### ISSUE THREAD

**Issue by Elekam** [catppuccin/tmux#127]
Hello, 

I am new to catppuccin and have been trying config 2 of the examples. But the icons dont render correctly. I tried installing multiple nerd fonts but they are still broken, no matter how much I resource or restart tmux.

Here my .tmux.conf:
```
set-option -sa terminal-overrides ",xterm*:Tc"

# Start windows and panes at 1, not 0
set -g base-index 1
setw -g pane-base-index 1

# Set srollback buffer to 50000 instead of 2000.
# CAUTION: Can take quite a bit of ram space if using lots of panes
set-option -g history-limit 50000

#Activate mouse scroll
set -g mouse on

# List of plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'
set -g @plugin 'christoomey/vim-tmux-navigator'
set -g @plugin 'catppuccin/tmux'

#Catpuccin config
set -g @catppuccin_window_left_separator "█"
set -g @catppuccin_window_right_separator "█ "
set -g @catppuccin_window_number_position "right"
set -g @catppuccin_window_middle_separator "  █"

set -g @catppuccin_window_default_fill "number"

set -g @catppuccin_window_current_fill "number"
set -g @catppuccin_window_current_text "#{pane_current_path}"

set -g @catppuccin_status_modules_right "application session date_time"
set -g @catppuccin_status_left_separator  ""
set -g @catppuccin_status_right_separator " "
set -g @catppuccin_status_right_separator_inverse "yes"
set -g @catppuccin_status_fill "all"
set -g @catppuccin_status_connect_separator "no"

# set -g @plugin 'github_username/plugin_name'
# set -g @plugin 'github_username/plugin_name#branch'
# set -g @plugin 'git@github.com:user/plugin'
# set -g @plugin 'git@bitbucket.com:user/plugin'

# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm'
```

![image](https://github.com/catppuccin/tmux/assets/43169119/043f5d51-2327-4a8d-946f-dabc46a2952e)

I have tried updating the plugins. I am using WSL Ubuntu.
I saw that apparently there was a bigger refactor of the repo, and that the examples might be outdated?
Thanks for any help!

**Comment by 89iuv** [comment]
This looks like a font issue. Try to use one of the nerd fonts: https://www.nerdfonts.com.

**Comment by Elekam** [comment]
Hey,

I did. I first installed 0xProto Nerd Font, but that didnt work after a restart of tmux. I then installed ProtoX Nerd Font Mono aswell as JetBrainsMonoNerdFontPropo-Regular, and then restarted my pc. But none of it seems to change anything, and I dont know why. Do you need any other information? Do you have an idea what this could be? Installing a font seems relatively simple, but maybe I missed something?

**Comment by 89iuv** [comment]
Did you also set the new font in your teminal?

**Comment by Elekam** [comment]
Now it works, thank you. For other people on Windows11 that might not know how to do this, I visited this [site](https://pureinfotech.com/change-font-face-windows-terminal/) yesterday and tried the mentioned key-shortcut Shift +  Ctrl + , (comma), which just opened the windows terminal settings.json in my vscode, with no font option to be found. I managed to get to the correct settings screen by clicking the small "arrow down" button next to the windows terminal tab, pressing "settings", then under "Profiles" select "Defaults" and you will see the option to change your font.

Thanks for your help! Sorry for bothering you.
